analysis_plan.md

```diff
@@ -0,0 +1,25 @@
+Comparison of SARS-CoV & SARS-CoV-2 Viruses from China
+------------------------------------------------------
+------------------------------------------------------
+
+1. Search literature for nucleotide conbinations that differ in SARS-CoV & SARS-CoV-2 viruses
+
+2. Decide which DNA nucleotide combinations add to the Dictionary
+
+3. Use ```function kmercount(sequence, k)``` to calculate the numerical difference of the nucleotide 
+
+conbinations of interest in the DNA of SARS-CoV & SARS-CoV-2 viruses
```

Comment:
I think it will be hard to specifically identify short stretches that have meaning in the literature. Typically, we use kmers for a gross overall sequence composition metric, or alignments for looking at functional sequencees in a specific location